spring-cloud-config-client/src/main/java/org/springframework/cloud/config/client/ConfigClientProperties.java

```diff
@@ -76,7 +76,7 @@
 	/**
 	 * The URI of the remote server (default http://localhost:8888).
 	 */
-	private String uri = "http://localhost:8888";
+	private String[] uri = {"http://localhost:8888"};
```

Comment:
seems to make sense to change the name to be `uris`

Reply:
Not sure. That would break all existing clients. I'd like to keep the configuration backwards compatible

Reply:
isnt that acceptable in a major release?

Reply:
In general yes. But if there's a backwards compatible way for this particular one, I'd think it would save us support issues in the future

Reply:
@spencergibb ur concern is with the name of property or the type?

Reply:
Name

Reply:
OK I will buy that